content/rest/push.textile

```diff
@@ -11,4 +11,80 @@ jump_to:
     - Platform support#platform-support
 ---
 
-<%= partial partial_version('general/push/_push_intro') %>
\ No newline at end of file
+<%= partial partial_version('general/push/_push_intro') %>
+
+h1. Push API reference
+
+inline-toc.
+  Push Admin:
+    - Methods:
+      - publish#publish
+    - Properties:
+      - deviceRegistrations#device-registrations
+      - channelSubscriptions#channel-subscriptions
+    - DeviceRegistrations:
+      - get#device-get-id
+      - list(params)#device-list
+      - save(DeviceDetails)#device-save
+      - remove#device-remove-id
+      - removeWhere(params)#device-remove-where
+    - PushChannelSubscriptions:
+      - get(PushChannelSubscription)#push-channel-sub-get
+      - list(params)#push-channel-sub-list
+      - listChannels(params)#push-channel-sub-list-channels
+      - save(PushChannelSubscription)#push-channel-sub-save
+      - remove(PushChannelSubscription)#push-channel-sub-remove
+      - removeWhere(params)#push-channel-sub-remove-where
+  Push Device:
+    - Methods:
```

Comment:
These don't exist in the REST API.  The REST API only offers admin functionality.